stdlib/marshal.pyi

```diff
@@ -1,5 +1,5 @@
 from _typeshed import ReadableBuffer
```

Comment:
```suggestion
from _typeshed import ReadableBuffer, SupportsRead, SupportsWrite
```

Reply:
Oh you just did that, thanks! GitHub's UI doesn't handle this well: the "Commit" button still shows up on the suggestion but clicking it does nothing. :)

Reply:
Already done ;)